book/scaling.md

```diff
@@ -0,0 +1,45 @@
+# Making Your Yii2 App Scalable
```

Comment:
On the other hand, it explicitly says "application". Did you mean "PHP code only"?